Task03-OnePieceWebsite/home-page.html

```diff
@@ -0,0 +1,217 @@
+<!-- 
+    All Rights Reserved
+    This software is property information of AKUREY S.A.
+    Contact us at contact@akurey.com
+    @summary short description for the file
+-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page</title>
+    <link href="./style/style.css" rel="stylesheet">
+    <link href='https://fonts.googleapis.com/css?family=Roboto Slab' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<script src="home-page.js"></script>
+<body>
+    <div class="navbar">
+        <section class="bar">
+            <button class="navbar-item" onclick="showMenu()">
+                <img id="menu-icon" src="./svg/menu.png" alt="menu">
+                <span class="material-icons" id="close-icon">close</span>
+            </button>
+            <img class="navbar-logo" src="./svg/One-Piece-Logo 1.png" alt="One-Piece-Logo">
+            <div class="navbar-item"></div>
+        </section>
+        <section class="navbar-menu">
+            <a class="navbar-option" href="/">Home</a>
+            <a class="navbar-option" href="/characters">Characters</a>
+            <a class="navbar-option" href="/islands">Islands</a>
+            <a class="navbar-option" href="/mist-objects">Mist Objects</a>
+            <div class="lang-option-container">
+                <button class="lang-option">
+                    <small class="navbar-option" id="lang-text">English</small>
+                    <img src="./svg/usd.svg" alt="usd">
```

Comment:
Let's try to be more specific with alt texts, these are important for accessibility in the future, I will use something like "United States flag icon"